docs/source/verticapylab_gs.rst

```diff
@@ -0,0 +1,141 @@
+.. _verticapylab_gs:
+
+=================
+VerticaPyLab
+=================
+
+The easiest way to try out VerticaPy is with `VerticaPyLab <https://github.com/vertica/VerticaPyLab>`_, a docker-based 
+JupyterLab environement designed to provide users with a powerful, easy-to-use development platform for VerticaPy. Capitalizing
```

Comment:
I am not sure if this is what we want to communicate about VerticaPyLab: "easy-to-use development platform for VerticaPy".

The primary purpose is not development. Rather it is to try out Vertica. And then we could also mention that it offers a NO-CODE approach to some of the VerticaPy functionalities. Additionally, the VerticaPy lessons which help users to learn the product. 

But the primary should be to try Vertica + GUI based functionalities